.eslintrc.js

```diff
@@ -15,7 +15,8 @@ module.exports = {
     "plugin:import/errors",
     "plugin:import/warnings",
     "plugin:jsx-a11y/recommended",
-    "plugin:security/recommended"
+    "plugin:security/recommended",
+    "airbnb-base/whitespace"
```

Comment:
Use Airbnb's widely used ruleset, but only the whitespace stuff (bc our other rules may not mesh well with Airbnb's, and I don't want to unilaterally change all the rules already declared below).